domains/client/domain-operator/src/fraud_proof.rs

```diff
@@ -323,15 +324,22 @@ where
                 total_extrinsics: local_trace_index - 1,
             };
             let finalize_block_call_data = Vec::new();
-
+            let inherent_data = get_inherent_data::<_, _, Block>(
+                self.consensus_client.clone(),
+                local_receipt.consensus_block_hash,
+                parent_header.hash(),
+                domain_id,
+            )
+            .await?;
```

Comment:
I'm not quite sure if we need to provide the `inherent_data` here, because the `extrinsics` are getting from the domain block body which supposes already include the inherent extrinsic, so if we provide the `inherent_data` it may cause duplicated inherent extrinsic.

Reply:
Nope, removed that infra completely. They wont be passed since that was always weird to me to pass them rather than using the proper inherent extrinsic route to derive those extrinsics.